src/Plugin/Derivative/OgLocalTask.php

```diff
@@ -42,7 +42,7 @@ class OgLocalTask extends DeriverBase implements ContainerDeriverInterface {
    */
   public function __construct(GroupTypeManagerInterface $group_type_manager, RouteProvider $route_provider) {
```

Comment:
The route provider service is replaced with a version that will rebuild (RouteProviderLazyBuilder) if routes used during installation. Not using an interface causes an error.
```suggestion
  public function __construct(GroupTypeManagerInterface $group_type_manager, RouteProviderInterface $route_provider) {
```